app/controllers/sops_controller.rb

```diff
@@ -56,6 +56,26 @@ def update
     end
   end
 
+  def dt_typeahead
```

Comment:
give a more descriptive name, e.g dynamic_table_typeahead, so that it is clearer in the future